index.js

```diff
@@ -1,44 +1,65 @@
 const express = require('express');
 const bodyParser = require('body-parser');
-const app = express()
+const cors = require('cors')
+const app = express();
+
+app.use(cors());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-const Game = require('./src/game.js')
+const Game = require('./src/Game.js')
+const ShipPosition = require('./src/ShipPosition.js')
+//const connection = require('./database/ConnectionMSSQL.js')
+const connection = require('./database/ConnectionMySQL.js')
+const ShipTypeBd = require('./database/ShipModel.js')
 
-app.get('/game',(req, res) => {
+app.get('/game',(req, res, next) => {
   Game.join(req.query.token)
   .then(game => {
       res.send(game)
     })
   .catch(error => console.error(error))
 })
 
-app.post('/game', (req, res) => {
+app.post('/game', (req, res, next) => {
   Game.create(req.body)
     .then(game => {
       res.send(game)
     })
     .catch(error => console.error(error))
 })
 
-app.listen(3000, () => {
-  console.log('Example app listening on port 3000!')
-})
-
 
-/*
-  POST /game
-  {
-    "rows":
-    "columns"
-  }
-
-  {
-    "gameId": "http://localhost:3000/game?token=asdfasdfw23",
-    "playerId": "asdfasdfasdf"
-  }
-*/
+app.post('/game/:gameId/player/:playerId/ship-positions', (req, res, next) => {
```

Comment:
this was supposed to be a PUT to the "board" resource

Reply:
It was changed in the last commit for:
app.put('/game/:gameId/player/:playerId/board ...